pkgs/development/compilers/openjdk/openjfx/15.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchFromGitHub, writeText, openjdk11_headless, gradleGen
+{ stdenv, lib, fetchFromGitHub, writeText, openjdk11_headless, gradle_5_6
```

Comment:
Why does the newer version of openjfx use an older gradle?

Reply:
Fixed in f42d55d6f1f

My original plan was to remove Gradle 4.